rules/audit_logging_rules.yaml

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 # rules:
+#   # Note: Please use organization id, folder number and project id
```

Comment:
Technically, there is no folder number.... so this will have to be `folder id`.
How about this:
```
# Note: please use the id of the resource (such as organization id, folder id,
# project id, etc.) when specifying the resource ids.
```
I think the confusion might be when people use `name`, instead of `id`.